### PR TITLE
Apply aria-hidden to control-bar when userinactive

### PR DESCRIPTION
**Pull request by gkatsev**
Defaults to aria-hidden:true because the controls are hidden by default.

The next step here is to try and prevent the control bar from hiding when it has keyboard focus.

### CODE REVIEW

**Comment by gkatsev** [comment]
And got it to not go inactive when focused inside the player. That was surprisingly easy.
Though, I do wonder whether we want to do that or not. I guess that we could make sure that when `<Esc>` is pressed, we would blur out of the player so it can go inactive or something.
@OwenEdwards thoughts?

File: src/js/player.js [hunk 1]
```diff
@@ -2256,6 +2256,11 @@ class Player extends Component {
           this.addClass('vjs-user-active');
           this.trigger('useractive');
         } else {
+          // if we are focused anywhere in the player, don't go inactive
+          if (player.el().contains(document.activeElement)) {
```
Comment (gkatsev) [inline src/js/player.js]: maybe this should be `player.controlBar.el().contains(document.activeElement)`?
And then `<Esc>` would focus the player div?

File: src/js/player.js [hunk 2]
```diff
@@ -2256,6 +2256,11 @@ class Player extends Component {
           this.addClass('vjs-user-active');
           this.trigger('useractive');
         } else {
+          // if we are focused anywhere in the player, don't go inactive
+          if (this.el().contains(document.activeElement)) {
```
Comment (gkatsev) [inline src/js/player.js]: maybe this should be `this.controlBar.el().contains(document.activeElement)`?
And then `<Esc>` would focus the player div?

**Comment by gkatsev** [comment]
This should help with #2156 but it doesn't currently fix the trying to tab back into the player part of that issue.

**Comment by gkatsev** [comment]
Talking this over with @OwenEdwards, we realized we want to do basically the opposite of what this PR does. When the controls go inactive, it should only be visual. The controls should still work for Screen Readers. I'll close this and approach this problem from that direction.

**Comment by OwenEdwards** [comment]
@gkatsev "still work for Screen Readers" __and keyboard-only users__. I'm going to keep hammering on that point, that these are two different groups of users (although they experience related accessibility issues). Honestly, you'll miss me when I don't need to mention it again ;-)

**Comment by gkatsev** [comment]
Yeah, it didn't feel right to not go inactive on focus but didn't understand exactly where I was going wrong. :D

**Pull request closed without merging**